.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -61,35 +61,25 @@ MUST: This issue cannot be closed until the mini-PM is written and its action it
 
 ## Summary
 
-<!--
-TIP: A few sentences summarizing the problem and impact.
--->
-
-<_YOUR_SUMMARY_>
+<_1-2 sentences summarizing the problem and root causes._>
 
-| Users affected                                   | Impact                                                                       |
-| ------------------------------------------------ | ---------------------------------------------------------------------------- |
-| <_Which users were affected? Roughly how many?_> | <_How were users affected? E.g. partial or complete loss of functionality?_> |
+## Impact
 
-## Root Causes
-
-1.
+- <_Which users were affected? Roughly how many?_>
+- <_How were users affected? E.g. partial or complete loss of functionality?_>
```

Comment:
Instead of a normal bulleted list, how about using checkboxes to make it a [task list](https://help.github.com/en/github/managing-your-work-on-github/about-task-lists)? Github's UI will show the issue as incomplete until all boxes are checked.

Reply:
Tried it and looks a bit weird since the actual content isn't actionable.